lib/node_modules/@stdlib/repl/lib/help_text.js

```diff
@@ -22,84 +22,87 @@
-	'    varsWorkspace()         List workspace variable names.',
-	'    loadWorkspace()         Load variables from a specified workspace.',
-	'    clearWorkspace()        Delete variables in a specified workspace.',
-	'    deleteWorkspace()       Delete a specified workspace.',
-	'    renameWorkspace()       Rename a specified workspace.',
-	'    currentWorkspace        Return the name of the current workspace.',
-	'',
-	'    themes()                List available color themes.',
-	'    addTheme(name,value)    Add a new color theme.',
-	'    deleteTheme(name)       Delete a specified color theme.',
-	'    renameTheme(old,new)    Rename a specified color theme.',
-	'    getTheme([name])        Return the current (or a specified) theme\'s color palette.',
-	'',
-	'    assignin()              Assign a value to a workspace variable.',
-	'    assignfrom()            Read in a value from another workspace.',
-	'',
-	'    evalin()                Evaluate an expression in a specified workspace.',
-	'',
-	'    rerun()                 Rerun previous commands.',
-	'',
-	'    require()               Import a module, JSON, or local file.',
-	'    rerequire()             Re-import a module, JSON, or local file.',
-	'    deeprerequire()         Re-import a module and its dependencies.',
-	'    load()                  Load and evaluate a JavaScript file line-by-line.',
-	'',
-	'    save()                  Save previous commands to a specified file.',
-	'    saveStart()             Start saving commands to a specified file.',
-	'    saveStop()              Stop saving commands.',
-	'',
-	'    presentationStart()     Start a REPL presentation.',
-	'    presentationStop()      Stop a REPL presentation.',
-	'',
-	'    userDoc()               Add user-defined documentation.',
-	'    clearUserDocs()         Clear user-defined documentation.',
-	'',
-	'    clearHistory()          Clear the REPL history.',
-	'',
-	'    clear()                 Clear the entire REPL screen and scrollback history.',
-	'    reset()                 Reset the REPL.',
-	'    quit()                  Exit the REPL.',
-	'',
-	'    aliases()               List namespace contents.',
-	'    alias2pkg()             Convert an alias to a stdlib package name.',
-	'    pkg2alias()             Convert a stdlib package name to an alias.',
-	'    alias2related()         List related aliases.',
-	'    pkg2related()           List related package names.',
-	'    isKeyword()             Assert whether a value is a reserved keyword/alias.',
-	'',
-	'    citation()              Print how to cite stdlib in publications.',
-	'    license()               Print license information.',
-	'    copyright()             Print copyright information.',
-	'    contributors()          List contributors.',
-	'    credits()               Print credits.',
-	'    donate()                Print donation information.',
+	'    help()                      Print help text.',
```

Comment:
This wasn't explicitly stated and was generally the case, but this pushes the lines past 80 characters. I suppose we could wrap. E.g., 

```
	'    getTheme([name])            Return the current (or a specified) theme\'s
	'                                color palette.',
	''
```